website/cue/reference/releases/0.23.0.cue

```diff
@@ -26,6 +26,7 @@ releases: "0.23.0": {
 	known_issues: [
 		"Vector shuts down when a configured source codec (`decoding.codec`) receives invalid data. Fixed in v0.23.1.",
 		"The `elasticsearch` sink doesn't evaluate templated configuration options like the `index` configuration before applying the `only_fields` and `except_fields` options, causing templates to fail to be evaluated if they used a field that was dropped. Fixed in v0.23.1.",
+		"The `datadog_traces` sink APM stats calculation does not correctly aggregate the stats in the way that is expected by the APM backend of Datadog, causing incorrect individual span metrics observed in the Datadog UI. Fixed in v0.25.2."
```

Comment:
```suggestion
		"The `datadog_traces` sink APM stats calculation does not correctly aggregate the stats in the way that is expected by the APM backend of Datadog, causing incorrect individual span metrics observed in the Datadog UI. Fixed in v0.25.2.",
```

As mentioned in the docs-check: https://github.com/vectordotdev/vector/actions/runs/3567742513/jobs/5995805639#step:14:31